Exercise_A/index.html

```diff
@@ -12,6 +28,31 @@
     />
   </head>
   <body>
-    <!-- CODE HERE -->
+    <nav class="navbar navbar-expand-lg navbar-light bg-link">
+      <div class="container-fluid">
+        <a class="navbar-brand" href="#"><img src="/Exercise_A/assets/bootstrap.png" alt=""  width="30" height="30"></a>
+        <div class="collapse navbar-collapse" id="navbarNav">
+          <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+            <li class="nav-item">
+              <a class="nav-link active text-primary" aria-current="page" href="#">Home</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link text-primary" href="#">Link</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+    <ul class="list-group">
+      <li class="list-group-item">Item I</li>
+      <li class="list-group-item">Item II</li>
+      <li class="list-group-item">Item III</li>
+      <li class="list-group-item">Item IV</li>
+      <li class="list-group-item">Item V</li>
+    </ul>
+    <button type="button" class="btn btn-primary">Primary</button>
+    <button type="button" class="btn btn-secondary">Secondary</button>
+    <button type="button" class="btn btn-success">Success</button>
+    <button type="button" class="btn btn-danger">Danger</button>
   </body>

```

Comment:
Great implementation of the bootstrap page, and nice work with your html code and structure!